apps/ledger-live-mobile/src/components/WalletTab/WalletTabNavigatorScrollManager.tsx

```diff
@@ -55,7 +55,7 @@ export default function WalletTabNavigatorScrollManager({
           if (
             scrollYValue !== null &&
             item.value &&
-            ((scrollYValue < headerHeight && scrollYValue >= 0) ||
+            (scrollYValue < headerHeight ||
```

Comment:
I'm uncertain of the reason this existed but this keeps the view sin sync when pulling to refresh. Is there another use case I may have broken doing this?

Reply:
@nparigi-ledger